docker/client.py

```diff
@@ -28,10 +28,14 @@
 from .auth import auth
 from .unixconn import unixconn
 from .ssladapter import ssladapter
-from .utils import utils, check_resource, update_headers
+from .utils import utils, check_resource, update_headers, kwargs_from_env
 from .tls import TLSConfig
 
 
+def from_env(**kwargs):
```

Comment:
I like the neatness too. It's always a tiny mental speed-bump when I have to learn a new class when starting out with a library, so having that removed is :+1: